fdb-relational-server/src/main/java/com/apple/foundationdb/relational/server/jdbc/v1/TransactionRequestHandler.java

```diff
@@ -82,9 +82,13 @@ public void onNext(final TransactionalRequest transactionRequest) {
                 final FRL.Response response = frl.transactionalExecute(transactionalToken, request.getSql(),
                         request.getParameters().getParameterList(), fromProto(request.getOptions()));
 
-                StatementResponse.Builder statementResponseBuilder = StatementResponse.newBuilder()
-                        .setResultSet(response.getResultSet())
-                        .setRowCount(response.getRowCount());
+                StatementResponse.Builder statementResponseBuilder = StatementResponse.newBuilder();
```

Comment:
creating temporary tables does not return a resultSet, so this was failing because the `Optional` underneath was not present.

Reply:
Created issue: #3475